### ISSUE TITLE
Example 22 uses termsOfUse

### ISSUE THREAD

**Issue by pchampin** [w3c/vc-data-model#776]
[Example 22](https://www.w3.org/TR/vc-data-model/#example-22-usage-of-the-termsofuse-property-by-a-holder) uses `termsOfUse` as a property of a veriable presentation.

However, in [the context](https://www.w3.org/2018/credentials/v1), `termsOfUse` is only defined in the local context of `VerifiableCredential`.

**Comment by msporny** [comment]
> [Example 22](https://www.w3.org/TR/vc-data-model/#example-22-usage-of-the-termsofuse-property-by-a-holder) uses `termsOfUse` as a property of a veriable presentation.
> 
> However, in [the context](https://www.w3.org/2018/credentials/v1), `termsOfUse` is only defined in the local context of `VerifiableCredential`.

Yes, this is a bug in the JSON-LD context, nice catch @pchampin!

Noting what the spec [says for ToU](https://www.w3.org/TR/vc-data-model/#terms-of-use):

"Terms of use can be utilized by an issuer or a holder to communicate the terms under which a verifiable credential or verifiable presentation was issued."

We always meant `termsOfUse` to be usable from within a presentation... we just didn't carry that through to the JSON-LD Context.

> cc @msporny for any thoughts on the substantiveness of modifying the VP context defn

Imma spec language lawyer here for a second:

1. We always meant `termsOfUse` to be usable from within a presentation.
2. The JSON-LD Context in the specification is non-normative.

So, technically speaking, we could update the JSON-LD Context in place and fix the bug with little chance that things could go wrong. 

*takes spec lawyer hat off*

That said, I don't think we should change the 2018 VC context and fix it... people are using that context in production. We should treat this like a substantive change and put it in the next JSON-LD Context for VCs. I suggest that even though it's not technically a substantive change to the 2018 context, it's still effectively a substantive change since it would affect implementation behavior (the term would go from being dropped to not being dropped during signing).

Instead, we should fix this in the next version of the VC context... noting that we're going to be making a bunch of other changes at the same time (like factoring the cryptosuites out).

**Comment by kdenhartog** [comment]
I've written a PR to remove this from the specification per @wyc option 1. We'll need to either update this issue afterwards and leave it open to track re-adding the text back in or we'll need to open another issue after it's been merged.

**Comment by TallTed** [comment]
@kdenhartog -- Since this is closed, please add a link here or there connecting this issue to the new one about putting the text back in.  The new issue should be tagged for version 2.0, if it isn't already.

**Comment by kdenhartog** [comment]
Thanks for the reminder @TallTed. I've gone and created a follow up issue to address this in V2 in #810
